gulp/styles.js

```diff
@@ -61,13 +61,16 @@ export function stylesAfterReplacementStream() {
 				config.dev.styles.stage :
 				3
 			),
-			preserve: true,
```

Comment:
Removing the global `preserve: true` is necessary as setting it overrides any individual options passed.